src/main/java/org/terasology/Sample/blocks/RomanColumnFamily.java

```diff
@@ -50,15 +88,112 @@ private Block getProperBlock(WorldProvider worldProvider, Vector3i location) {
             }
         }
         if (sides.contains(Side.BOTTOM) && !sides.contains(Side.TOP)) {
```

Comment:
I'm still going to argue this is overly verbose and defeats the purpose of using a map in the first place.